src/pageObject.js

```diff
@@ -1,16 +1,41 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
```

Comment:
It seems like you're returning a string in the getter method for the logo. Usually, in a page object model, we would return an actual element or a promise of an element from a getter, not a string. Are you planning to implement this later?